modules/p2/ding_serendipity_lists/ding_serendipity_lists.module

```diff
@@ -361,6 +361,12 @@ function ding_serendipity_lists_author_from_lists_serendipity_add($context) {
 
     $query = implode(" AND ", $terms);
 
+    // Limit to same language to reduce signal to noise.
+    $lang = $ting_entity->getLanguage();
+    if (!empty($lang)) {
+      $query = "(" . $query . ") AND facet.language=" . $ting_entity->getLanguage();
```

Comment:
Nitpicking: Once you have extracted a value for an object there it no reason not to reuse it for tests and string concatenation.